aspnetcore/blazor/javascript-interoperability/index.md

```diff
@@ -18,6 +18,16 @@ This overview article covers general concepts. Further JS interop guidance is pr
 * <xref:blazor/js-interop/call-javascript-from-dotnet>
 * <xref:blazor/js-interop/call-dotnet-from-javascript>
 
+## Interaction with the Document Object Model (DOM)
+
+Only mutate the Document Object Model (DOM) with JavaScript (JS) when the object doesn't interact with Blazor. Blazor maintains representations of the DOM and interacts directly with DOM objects. If JS interop mutates an element or its content and Blazor attempts to apply diffs to the element, the diffs won't match the DOM and undefined behavior results. Arbitrary behaviors may merely interfere with the presentation of elements or their behavior but may also introduce security risks to the app or server.
```

Comment:
> If JS interop mutates an element or its content and Blazor attempts to apply diffs to the element, the diffs won't match the DOM and undefined behavior results

Perhaps, this should say 

If an element or its content that is rendered by Blazor is modified externally, e.g. using JavaScript including JS Interop, the DOM may no longer match Blazor's internal representation for it and would result in undefined behavior.